modules/admission-webhooks-about.adoc

```diff
@@ -45,3 +45,8 @@ Some common webhook admission plugin use cases include:
 * Defining tolerations that enable taints to qualify which pods should be scheduled on a node.
 //Future xref - * xref:../nodes/pods/nodes-pods-priority.adoc#admin-guide-priority-preemption-names_nodes-pods-priority[Pod priority class validation].
 * Pod priority class validation.
+
+[NOTE]
+====
+By default, in {product-title} webhooks timeout after 13 seconds.
```

Comment:
The language "by default" implies that the default is 13 seconds but does not tell the user that they can't change this value (which seems to be what the bug is saying - unless I am not understanding the issue fully). It might be more clear to reword. Suggestion: "The default webhook timeout value in {product-title} is 13 seconds and cannot be changed." (or something similar)

If you decide not to change the wording, when used as a verb, time out is two words (IBM Style).

Reply:
Yes. A user can't change the webhook timeout value -  that's correct

Addressed

Reply:
@kcarmichael08 yeah, I think your wording is better. The hard upper limit is 13 seconds. So you can configure 10s and it will timeout after 10s, but if you configure 30s, then it will always timeout after 13s. 
So maybe:

Suggestion: "The default webhook upper timeout value in {product-title} is 13 seconds and cannot be increased even if higher timeout is configured." (or something similar)

Reply:
I will address this change @vwalek